app/deployments/tests/test_tasks.py

```diff
@@ -1,4 +1,5 @@
 from unittest.mock import patch
+import pytest
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'pytest'](https://app.codacy.com/manual/gmri/buoy_barn/pullRequest?prid=4987082)